mlfinlab/labeling/raw_return.py

```diff
@@ -0,0 +1,43 @@
+"""
+Most basic form of labeling based on raw return of each observation relative to its previous value. User can specify
+percentage or logarithmic, numerical or categorical labels, and number of steps back to calculate the raw return from.
+"""
+
+import warnings
+import numpy as np
+
+
+def raw_return(price, binary=False, logarithmic=False, lookback=1):
+    """
+    Raw returns labeling method.
+
+    This is the most basic and ubiquitous labeling method used as a precursor to almost any kind of financial data
+    analysis or machine learning. User can specify percentage/logarithmic returns, numerical/binary labels, and
```

Comment:
Not percentage/logarithmic, they are both in percentage terms. Its `simple returns or log returns` not percentage.

Reply:
Fix this to your notebook as well.